.release/ci.hcl

```diff
@@ -150,6 +150,15 @@ event "verify" {
   }
 }
 
+event "fossa-scan" {
+  depends = ["verify"]
+  action "fossa-scan" {
+    organization = "hashicorp"
+    repository = "crt-workflows-common"
+    workflow = "fossa-scan"
+  }
+}
+
```

Comment:
This fails our lint check. If you run `hclfmt` on it, it'll pass.

Reply:
Thank you! This is fixed in the latest commit. 

The report links in fossa are available in the `fossa-scan` job- this [short recording](https://recordit.co/LhqLDmkMKz) walks through how to access find the link and view the issues in fossa. Fossa access _should_ be available to everyone via Okta, but if you don't see the fossa tile on hashicorp.okta.com, sending a ticket to helpdesk will make that available to you/your team.